5-predict_subtype_reconstructed_data.R

```diff
@@ -74,25 +74,33 @@ for (seed in filename.seeds) {
       
       # read in reconstructed data from current platform and reconstruction 
       # method
-      file.identifier <- paste(plt, rcn, seed, sep = "_")
+      file.identifier <- paste(rcn, plt, seed, sep = "_")
       recon.rds <- recon.files[grep(file.identifier, recon.files)]
       recon.list <- readRDS(recon.rds)
       
       # get confusionMatrix objects
-      plt.list[[rcn]] <- PredictReconDataWrapper(train.list = train.list,
-                                                 recon.list = recon.list,
-                                                 sample.df = sample.df)
+      plt.list[[rcn]] <- PredictWrapper(train.model.list = train.list,
```

Comment:
Kappa is always calculated as part of a `confusionMatrix` (by using `confusionMatrix` from `caret`). `GetCM` (first function in `train_test_functions.R`) is where the choice to return either the whole confusionMatrix or just the Kappa statistic is handled (see lines 32-39).

What I was previously doing (and would not require two runs of `PredictWrapper` in the loop) was having the `confusionMatrix` returned (`return.kap = FALSE`) was using a series of nested `lapply` which we deemed difficult to follow in #7. See [this exchange](https://github.com/greenelab/RNAseq_titration_results/pull/7#issuecomment-270644430), specifically.